osquery/dispatcher/dispatcher.h

```diff
@@ -24,6 +24,18 @@
 // paths for their includes. Unfortunately, changing include paths is not
 // possible in every build system.
 // clang-format off
+#ifndef OSQUERY_THRIFT_LIB
```

Comment:
This was messing up my linter, because the macros weren't defined. These are always set to these values anyway in CMake anyway, so this should have no effect, other than making statically analyzing the files without our build system a bit more accurate.